content/en/serverless/distributed_tracing/_index.md

```diff
@@ -2,48 +2,123 @@
 
-### Live Search
+If you are already tracing your serverless functions with X-Ray and want to continue using X-Ray, you can [install our AWS X-Ray integration][12].
 
-{{< img src="tracing/live_search/livesearchmain.gif" alt="Live Search" >}}
+*Looking to trace through serverless resources not listed above? Open a feature request [here][9].*
+
+#### Ruby
+
+The Datadog Lambda Library and tracing libraries for Ruby support:
+- Automatic correlation of Lambda logs and traces with trace ID and tag injection.
+- Tracing HTTP requests invoking downstream Lambda functions or containers.
+- Tracing dozens of additional out-of-the-box [Ruby][13] libraries.
+
+You can trace your serverless functions in Datadog with [Datadog's tracing libraries][8] or by [installing our AWS X-Ray integration][12]. If you are using [Datadog's tracing libraries][8], and need to connect Lambda function traces across AWS managed services, we recommend you augment your traces by configuring _both_ AWS X-Ray and Datadog APM tracing libraries as described [here][10].
+
+*Looking to trace through serverless resources not listed above? Open a feature request [here][9].*
+
+#### Go
+
+The Datadog Lambda Library and tracing libraries for Go support:
+- Manual correlation of Lambda logs and traces with trace ID and tag injection.
+- Tracing HTTP requests invoking downstream Lambda functions or containers.
+- Tracing dozens of additional out-of-the-box [Go][14] libraries.
+
+You can trace your serverless functions in Datadog with [Datadog's tracing libraries][8] or by [installing our AWS X-Ray integration][12]. If you are using [Datadog's tracing libraries][8], and need to connect multiple Lambda function traces in event-driven architectures, we recommend you augment your traces by configuring _both_ AWS X-Ray and Datadog APM tracing libraries as described [here][10].
+
+*Looking to trace through serverless resources not listed above? Open a feature request [here][9].*
+
+#### Java
+
+The Datadog Lambda Library for Java supports manual correlation of Lambda logs and AWS X-Ray traces with trace ID and tag injection. Datadog's tracing library for Java is currently in [beta][15]. We recommend you configure tracing for your Java serverless applications by [installing our AWS X-Ray integration][12].
 
-The APM Live Search gives you the ability to search all ingested spans using any tag in real-time for the past 15 minutes. It displays spans as soon as they are sent by the Datadog agent and before they are indexed by Datadog.
+*Have feedback on the Datadog's tracing libraries for Java Lambda functions? Make sure to check out discussions going on in the <a href="https://datadoghq.slack.com/archives/CFDPB83M4">#serverless</a> channel in the <a href="https://chat.datadoghq.com/">Datadog Slack community</a>.*
```

Comment:
Not sure if it's a big deal, but why not use the markdown link syntax instead of the html?